Python/ceval.c

```diff
@@ -1865,7 +1865,14 @@ _PyEvalFramePushAndInit_Ex(PyThreadState *tstate, _PyStackRef func,
     PyObject *kwnames = NULL;
     _PyStackRef *newargs;
     PyObject *const *object_array = NULL;
+    #if defined(__GNUC__)
```

Comment:
```suggestion
    #if defined(__GNUC__) && !defined(__clang__)
```
clang does not feature `-Wmaybe-uninitialized`. [^1]

[^1]: It does understand `-Wuninitialized`, but we don't have to care about clang here ...